File: src/Services/User/User.Domain/Entities/Loan.cs
using User.Domain.Common;

namespace User.Domain.Entities
{
    public class Loans : EntityBase
    {
        public decimal Amount { get; set; }
        public string Status { get; set; }
        public string OrganizationCode { get; set; }
        public string? OrganizationName { get; set; }
        public string AdharNumber { get; set; }
        public DateTime LoanDate { get; set; }
        public string LoanBorrower { get; set; }
        public string LoanType { get; set; }
        public string? RepaymentStatus { get; set; }
         public string? Remarks { get; set; }
        public string? SecurityReports { get; set; }
        public string? VehicleNo { get; set; }
        public string?  Suretyholder1 { get; set; }
        public string?  Suretyholder1Adhar { get; set; }
        public string?  Suretyholder2 { get; set; }
        public string?  Suretyholder2Adhar { get; set; }
    }
}

File: src/WebApps/clr-portal/src/app/loans/model/loan.ts
export interface Loan {
  amount: number;
  status: string;
  organizationCode: string;
  organizationName: string;
  adharNumber: string;
  loanDate: Date;
  loanBorrower: string;
  loanType: string;
  repaymentStatus: string;
  remarks:string;
  id: number;
  securityReports:string;
  vehicleNo:string;
  suretyholder1Adhar:string;
  suretyholder2Adhar:string;
  suretyholder1:string;
  suretyholder2:string;
}

export function compareLoans(c1: Loan, c2: Loan) {
  const compare = c1.id - c2.id;

  if (compare > 0) {
    return 1;
  } else if (compare < 0) {
    return -1;
  } else return 0;
}
